apps/src/templates/teacherNavigation/LessonMaterialsContainer.tsx

```diff
@@ -68,6 +71,7 @@ const createDisplayName = (lessonName: string, lessonPosition: number) => {
 const LessonMaterialsContainer: React.FC = () => {
   const loadedData = useLoaderData() as LessonMaterialsData | null;
   const lessons = useMemo(() => loadedData?.lessons || [], [loadedData]);
+  const unitNumber = useMemo(() => loadedData?.unitNumber || 0, [loadedData]);
```

Comment:
In the future, this `||0` will likely be `|| 1` for stand alone units.  Right now the `||0` helps raise issues with the unitNumber data not pulling through correctly.